### PR TITLE
Leaderboard Project Setup

### PR DESCRIPTION
**Pull request by kusiLaw**
This pull request  creates the basic structure of Leaderboard project  using Gitflow, webpack, and  ES6 modules, with import and export.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+     </section>
+     <section class = "add-scores">
+      <h2>Add your scores</h2>
+      <form action="#" id="add-form">
+       <input type="text" name="name" id="p-name" placeholder="Your name">
+       <input type="number" name="score" id="scores" placeholder="Your name">
+      <button type="Submit" id="add-btn">Submit</button>
+      </form>
+
+     </section>
```
Comment (codecaiine) [inline src/index.html]: - [ ] Good job so far project is very well implemented :+1: But please make your project match with the wireframe as required you can see some differences with your submit button position and also your displaying list border. Please fix it  cause as a developer is very important to pay attention :smile:  
Check the screenshots below :arrow_down: 

#### Your design
Screenshot from 2022-08-30 15-59-24

#### Original wireframe
Screenshot from 2022-08-30 15-59-18

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+### Show your support
+Give a star :star: if you like this project!
+
+
+### Acknowledgments
+Thanks to Microverse
+
+### 📝 License
+This Project is <a href ="https://opensource.org/licenses/MIT">MIT </a> licensed
```
Comment (codecaiine) [inline README.md]: - [ ] I noticed that you have a license which is perfect :+1:  but the link to your license is not the correct one :cry:  Please add the correct link or path to your customized licence in order to make your readme file more professional :smile:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @kusiLaw , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Good use of gitflow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @kusiLaw, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**